impeller/entity/geometry/geometry.h

```diff
@@ -48,6 +48,16 @@ ComputeUVGeometryCPU(
     Size texture_coverage,
     Matrix effect_transform);
 
+/// @brief Computes geometry and UV coordinates for a rectangle to be rendered.
+///
+/// UV is the horizontal and vertical axes within the texture.
```

Comment:
I think you mean the horizontal and vertical coordinates within the texture.

An axis is a direction, not a position.

Reply:
Done!